src/Sulu/Bundle/WebsiteBundle/Controller/WebsiteController.php

```diff
@@ -72,18 +72,22 @@ protected function renderStructure(
 
             // if not preview enable cache handling
             if (!$preview) {
-                // mark the response as either public or private
-                $response->setPublic();
-
-                // set the private and shared max age
-                $response->setMaxAge(240);
-                $response->setSharedMaxAge(960);
-
-                // set reverse-proxy TTL (Symfony HttpCache, Varnish, ...)
-                $response->headers->set(
-                    HttpCache::HEADER_REVERSE_PROXY_TTL,
-                    $response->getAge() + intval($structure->getCacheLifeTime())
-                );
+                if (!$this->getRequest()->isMethod('GET')) {
+                    $response->setPrivate();
```

Comment:
This is not disabling the cache, just enabling the private cache on the client side. You should probably just check for get and execute the current else part then. However, @dantleech has just move this functionality to a different handler in a PR being merged soon, so be careful that this change is not lost at all.